shell/browser/api/atom_api_cookies.cc

```diff
@@ -139,6 +139,17 @@ void FilterCookieWithStatuses(const base::Value& filter,
                 net::cookie_util::StripStatuses(list));
 }
 
+// Read date property from the dictionary.
+base::Time GetTimeFromDict(const base::DictionaryValue& dict,
```

Comment:
nit: it would be a little more flexible if this fetched a time from `base::Optional<double>` rather than from a dictionary.